lib/creditsafe/client.rb

```diff
@@ -86,6 +91,15 @@ def invoke_soap(message_type, message)
       notification_payload[:response] = response.body
     rescue => raw_error
       processed_error = handle_error(raw_error)
+
+      if processed_error.class == AccountError
+        retries -= 1
```

Comment:
Doesn't this logic mean that you'll make 3 requests in total, rather than 3 retries? (I could be going bad, I find this stuff really hard to reason about in my head...)